bcm283x/gpio.go

```diff
@@ -1331,17 +1320,20 @@ func (d *driverGPIO) String() string {
 }
 
 func (d *driverGPIO) Prerequisites() []string {
-	return nil
+	return []string{"ioctl-gpio"}
 }
 
 func (d *driverGPIO) After() []string {
-	return []string{"sysfs-gpio"}
+	return nil
```

Comment:
The driver used to be able to work even if sysfs was not available. This happened on obscure distros. I think I'd prefer to have ioctl-gpio be here instead of Prerequisites. So I'd prefer here to be:

```
return []string{"ioctl-gpio", "sysfs-gpio"}
```

because otherwise people with older distro will have issues.

Reply:
It doesn't work consistently that way. If you want to tell me why, I'll see what I can do.

Reply:
It's okay, I think we can just enforce this.